ck.sh

```diff
@@ -58,36 +58,35 @@ case "$COMMAND" in
 
 		case $yn in
 			[Yy]*)
-				# Copy custom skin to ckeditor-dev
-				cp -r skins/moono-lexicon ckeditor-dev/skins/moono-lexicon
-
-				# Copy custom plugins to ckeditor-dev
-				cp -r plugins ckeditor-dev
-
-				# Copy skin files for plugins
-				pluginsWithSkins=$(find ckeditor-dev/plugins -maxdepth 2 -mindepth 2 -type d -name skins)
-				for plugin in $pluginsWithSkins; do
-					pluginDir=$(dirname "$plugin")
-					pluginName=$(basename "$pluginDir")
-					pluginSkinPath="skins/moono-lexicon/plugins/$pluginName/skins/moono-lexicon"
-					if [[ -d "$pluginSkinPath" ]]; then
-						cp -r "$pluginSkinPath" "$pluginDir/skins"
-					fi
-				done
+				# Clean and make a temporal copy of ckeditor-dev to work with
+				rm -rf temp
+				cp -r ckeditor-dev temp
```

Comment:
I feel a bit queasy about copying an entire Git submodule (in practice, at least, the `.git` directory is not a real directory, but a "gitlink" file, so at least we're not copying that). Right now, it doesn't have anything that could be "tricky" (eg. symlinks), but it might in the future. We're talking about > 100MB of stuff.

Reply:
Yes, sounds like a good approach for now.